docker/iml-stats.dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM rust-iml-base as builder
+
+FROM ubuntu
```

Comment:
Shouldn't we use centos7? Everything else will be using it once the timer patch lands.

Reply:
yeah, but we can address once the timer patch has landed.